.github/workflows/README.md

```diff
@@ -119,7 +119,7 @@ Concurrency groups are a way of making sure that no more than one Actions run is
 
 ```
 concurrency:
-  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.event.pull_request.head.label || github.sha || github.head_ref || github.ref }}-${{ github.event.schedule || github.event.comment.id || github.event.sender.login}}'
+  group: '${{ github.workflow }} @ ${{ github.event.issue.number || github.sha || github.head_ref || github.ref }}-${{ github.event.schedule || github.event.comment.id || github.event.sender.login }}'
```

Comment:
@damccorm This is actually not quite correct - we keep using the `github.event.pull_request.head.label` condition but only in case there is a `pull_request_target` event trigger. For instance, all `PreCommit` workflows have this event and the condition should be specified (e.g. https://github.com/apache/beam/blob/master/.github/workflows/beam_PreCommit_Go.yml#L50).
I think it makes sense to add a comment bellow describing this.

Reply:
@andreydevyatkin would you mind putting up a PR to do this?